lib/modules/filteReddit.js

```diff
@@ -746,9 +753,17 @@ module.always = function() {
 	Menu.addMenuItem(nsfwSwitch, onClickSwitch);
 };
 
+let filterline;
+export let forceExpandoCompletion = false;
+export let expandoCompletionCallback;
```

Comment:
Similar thing here, we shouldn't put the burden of null-checking on the caller nor should showImages know about the internals of this module. Instead, have showImages export a function to register (and unregister, if necessary) callbacks.